### DISCUSSION TITLE
amvstrm is back...

### DISCUSSION THREAD

**Discussion by NYT92** [amvstrm/amvstrm#51] [Announcements]
We recently pushed the new update for API and website open to the public for all. We are still working on the new docs website to add more information about the usage of the API and more info about other stuff. 

Speaking of the new API, last month we got hit with the worst DDoS ever, causing the API to exceed the free usage limit. We do not host our API on an expensive server, please host yourself if you needed a giant ass request to your server. The new protection and new rate-limit will be reduced to 60 per min. If that continues to happen, we will not serve any more free APIs.

More info will be out soon about other stuff....

**Comment by JammyVibez** [comment]
I'm really happy
